### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by zebratrois**
An improved french translation as the current one is plain wrong with some strings and not quite grammatically correct with the others...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 vendor/
 bin/
 composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
+.idea/
```
Comment (norzechowicz) [inline .gitignore]: .idea should not be included in project .gitignore file. You can setup in your global .gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (zebratrois) [inline .gitignore]: Didn't know about that, I've reverted the commit...

**Comment by norzechowicz** [comment]
could you please rebase it?

**Comment by zebratrois** [comment]
Should be ok now (if i'm not mistaken as i'm not qui a git expert...)

**Pull request closed without merging**